### DISCUSSION TITLE
有试过blazor webassembly可以发布到微信小程序嘛？

### DISCUSSION THREAD

**Discussion by srxqds** [dotnetcore/BootstrapBlazor#4077] [Q&A] [answered]

**Comment by bb-auto** [comment]
@srxqds Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@srxqds 我这边主营是设备上使用 blazor 模式是 ssr 没有微信小程序这方便的实战案例

**Comment by srxqds** [comment]
ok,感谢

**Reply by ArgoZhang** [reply]
大佬不需要关闭这里是讨论区，方便后面的人搜索，点击回答下方的那个小对钩按钮即可，标记为答案

**Comment by srxqds** [comment]
ok，不知道有没有大佬尝试过在微信小程序里面使用blazor wasm

**Reply by densen2014** [reply, marked as answer]
需要企业版才能打开webview, 才能用外部url, 能用外部url了你用ssr, wasm, vue都没关系了, 不然只能用微信小程序那个框架搞. 如果帮到你了,点一下make as answer给更多的人看到

**Reply by srxqds** [reply]
这里说的是不支持的，https://developers.weixin.qq.com/community/develop/doc/00086483c90ff052d0706559466000 
不知道是不是同一个问题
不过我们试了下，是可以的，感觉是可以搞的，奇怪的是为啥没人弄

**Reply by densen2014** [reply]
> 这里说的是不支持的，https://developers.weixin.qq.com/community/develop/doc/00086483c90ff052d0706559466000 
> 
> 不知道是不是同一个问题
> 
> 不过我们试了下，是可以的，感觉是可以搞的，奇怪的是为啥没人弄

试试就知道了。听说过微信要推wasm

**Reply by srxqds** [reply]
是了，是没问题的
